controllers/component_build_controller_common.go

```diff
@@ -88,6 +88,29 @@ func getGitProvider(component appstudiov1alpha1.Component) (string, error) {
 	return gitProvider, err
 }
 
+func (r *ComponentBuildReconciler) GetPipelineFromComponentAnnotation(ctx context.Context, component *appstudiov1alpha1.Component) *tektonapi.PipelineRef {
```

Comment:
already existing function is called : GetPipelineForComponent

I'd rather keep my name, because when we will remove build pipeline selector, I want this function return only bundle & name instead of ref